### ISSUE TITLE
Accessibility: Selling: Focus lost upon dismissing 

### ISSUE THREAD

**Issue by aganeshalingam** [eBay/ebayui-core#941]
Focus lost from original element, after closing dialog.

**Comment by ianmcburnie** [comment]
@aganeshalingam The button is decoupled from the dialog. It's up to your app logic to set the focus.

**Comment by ianmcburnie** [comment]
Closing as invalid.

**Comment by ianmcburnie** [comment]
Done in 5.0.0 branch.
